static/app/views/releases/detail/overview/issues.tsx

```diff
@@ -284,6 +457,24 @@ const OpenInButtonBar = styled(ButtonBar)`
   }
 `;
 
+const StyledButtonBar = styled(ButtonBar)`
+  grid-template-columns: repeat(4, 1fr);
+  ${ButtonLabel} {
+    white-space: nowrap;
+    grid-gap: ${space(0.5)};
+    span:last-child {
+      color: ${p => p.theme.issueCountNotActiveText};
+    }
```

Comment:
We do not need to do it now, but the preferred way in the future would be to define StyledQueryCount instead of nested selectors.